### PR TITLE
Add tvOS support

### PR DESCRIPTION
**Pull request by sbrighiu**
;) Should be enough for tvOS support. @fethica

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,12 +31,14 @@ To run the example project, clone the repo, and run `pod install` from the Examp
...
+- [x] Support for tvOS
 - [ ] Support for Audio Taps
 - [ ] Support for Audio Recording
 
 ## Requirements
-- macOS 10.12 +
-- iOS 10 +
+- macOS 10.12+
+- iOS 10.0+
+- macOS 10.0+
```
Comment (misteral) [inline README.md]:
```suggestion
- tvOS 10.0+
```

Comment (misteral) [inline README.md]: @sbrighiu misspelling

**Comment by fethica** [comment]
Hey @sbrighiu, thanks for adding the tvOS support, everything is working, I will do some project adjustment and release the version 0.1.15. Cheers!

**Comment by sbrighiu** [comment]
;)

**Comment by fethica** [comment]
Done! [0.1.15](https://github.com/fethica/FRadioPlayer/releases/tag/0.1.15) is out 🚀

**Pull request merged**